app/assets/javascripts/whole_pot_calculator.js

```diff
@@ -0,0 +1,212 @@
+(function() {
+  'use strict';
+
+  function WholePotCalculator(income, pot) {
```

Comment:
Maybe `TakeWholePotCalculator` as the slug for the equivalent content is `take-whole-pot`?